alerter.c

```diff
@@ -21,6 +21,7 @@ void alertInCelcius(float farenheit) {
         // Add a test below to catch this bug. Alter the stub above, if needed.
         alertFailureCount += 0;
     }
+    assert(celcius<returnCode);
 }
 
 int main() {

```

Comment:
Test code and production code needs to be maintained separately in different files for readability and maintainability.